File: AppFlix higor-hugo padrao/AppFlix/app/src/main/java/br/com/senaijandira/room/Producao.java
package br.com.senaijandira.room;

/**
 * Created by sn1041520 on 27/03/2018.
 */

public class Producao {

    private int id;
    private String titulo;
    private String sinopse;
    private String link;
    private Double avaliacao;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getSinopse() {
        return sinopse;
    }

    public void setSinopse(String sinopse) {
        this.sinopse = sinopse;
    }

    public String getLink() {
        return link;
    }

    public void setLink(String link) {
        this.link = link;
    }

    public Double getAvaliacao() {
        return avaliacao;
    }

    public void setAvaliacao(Double avaliacao) {
        this.avaliacao = avaliacao;
    }
}

File: app meireles/apiappfix/Producao.php
<?php 
class Producao {

	private $id;
	private $titulo;
	private $sinopse;
	private $caminhoVideo;
	private $caminhoFoto;
	private $avaliacao;

	public function getId(){
		return $this->id;
	}
	public function setId($id){
		$this->id = $id;
	}

	public function getTitulo(){
		return $this->titulo;
	}
	public function setTitulo($titulo){
		$this->titulo = $titulo;
	}

	public function getSinopse(){
		return $this->sinopse;
	}
	public function setSinopse($sinopse){
		$this->sinopse = $sinopse;
	}

	public function getCaminhoVideo() {
		return $caminhoVideo;
	}
	public function setCaminho($caminhoVideo){
		$this->caminhoVideo = $caminhoVideo;
	}

	public function getCaminhoFoto() {
		return $caminhoFoto;
	}
	public function setCaminho($caminhoFoto){
		$this->caminhoFoto = $caminhoFoto;
	}

	public function getAvaliacao(){
		return $this->avaliacao;
	}
	public function setAvaliacao($avaliacao){
		$this->avaliacao = $avaliacao;
	}


}
?>